Library/Formula/hue.rb

```diff
@@ -0,0 +1,99 @@
+class Hue < Formula
+  desc "Hue is a Web interface for analyzing data with Apache Hadoop."
+  homepage "http://gethue.com/"
+  url "https://github.com/cloudera/hue/archive/release-3.8.1.tar.gz"
+  sha256 "582777f567b9f4a34e4ce58cfd5ed24aff15f3e02f193e7990329d8cb0161a8a"
+
+  depends_on :java
+  depends_on :mysql
+
+  depends_on "gmp"
+  depends_on "maven" => :build
+  depends_on "openssl"
+
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!